puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/hammer-cli-katello-master-release.groovy

```diff
@@ -0,0 +1,4 @@
+def owner_repo = 'Katello/hammer-cli-katello'
```

Comment:
Not needed right now, but we've renamed various repos (foreman_docker) to match their gem name. Probably good to have a similar look at hammer git repos. It'd allow us to introduce a `gem_name` variable and derive `gemspec` and `package_name` based on it. Again, future improvement :)

Reply:
yeah, hammer-cli-* bits could benefit from having their repos named to be the same as their packages/gems.